### PR TITLE
Rl training

### PR DESCRIPTION
**Pull request by sanagno**
- Code to run RL
- Custom PPO trainer with correct formatting for RM model
- Change imports until they work
- Some refactoring

### CODE REVIEW

File: model/model_training/utils/ppo_utils.py
Comment (theblackcat102) [inline model/model_training/utils/ppo_utils.py:1]: Any reason why there's a patch of used transformer code not removed?

Comment (sanagno) [reply model/model_training/utils/ppo_utils.py:1]: yes, that is on me, will be removed

**Review by Aayog — APPROVED** [review]

**Comment by sanagno** [comment]
@theblackcat102 @dvruette if everything looks good we can merge

File: model/model_training/tests/test_datasets.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 import pytest
 from model_training.custom_datasets import QA_DATASETS, SUMMARIZATION_DATASETS, get_one_dataset
 from model_training.custom_datasets.dialogue_collator import DialogueDataCollator
-from model_training.utils import get_tokenizer
+from model_training.utils.utils import get_tokenizer
```
Comment (andreaskoepf) [inline model/model_training/tests/test_datasets.py:6]: Personally I don't like utils.utils .. could be solved via __init__.py  in utils dir ... but no reason not to merge this now.

**Review by andreaskoepf — APPROVED** [review]
GOOOOO! :-) .. please let's try to create smaller PRs in the future.

**Pull request merged**